src/engraving/dom/types.h

```diff
@@ -249,9 +249,9 @@ enum class AccidentalType {
     SHARP_31,
     FLAT_53,
     SHARP_53,
-    //EQUALS_ALMOST,
-    //EQUALS,
-    //TILDE,
+    EQUALS_ALMOST,
+    EQUALS,
+    TILDE,
```

Comment:
Do you by chance still remember why these were commented out initially?

Reply:
No, I don't recall

Reply:
Probably because I didn't know which cent offsets to use